### PR TITLE
License

### PR DESCRIPTION
**Pull request by robertamarton**
[TRAFODION-1642] Release 1.3.0 copyrights and license files are incorrect, part 2

### CODE REVIEW

**Comment by svarnau** [comment]
Hmm, ODBC driver installation failed. The test script does this:

  cd $WORKSPACE/clients
  tar xvfz TRAF_ODBC_Linux_Driver_64.tar.gz
  echo ""
  cd PkgTmp
  ./install.sh <<-END_ODBC 
YES
$WORKSPACE/clients/odbc
$WORKSPACE/clients/odbc
$WORKSPACE/clients/odbc
$WORKSPACE/clients/odbc
END_ODBC

**Comment by robertamarton** [comment]
I’ll check it out.



   Roberta



*From:* Steve Varnau [mailto:notifications@github.com]
*Sent:* Friday, December 4, 2015 5:09 PM
*To:* apache/incubator-trafodion <incubator-trafodion@noreply.github.com>
*Cc:* Roberta Marton <roberta.marton@esgyn.com>
*Subject:* Re: [incubator-trafodion] License (#209)



Hmm, ODBC driver installation failed. The test script does this:

cd $WORKSPACE/clients
tar xvfz TRAF_ODBC_Linux_Driver_64.tar.gz
echo ""
cd PkgTmp
./install.sh <<-END_ODBC
YES
$WORKSPACE/clients/odbc
$WORKSPACE/clients/odbc
$WORKSPACE/clients/odbc
$WORKSPACE/clients/odbc
END_ODBC

—
Reply to this email directly or view it on GitHub
<https://github.com/apache/incubator-trafodion/pull/209#issuecomment-162121337>
.

File: core/conn/security_dll/LICENSE [hunk 1]
```diff
@@ -1,127 +0,0 @@
-
```
Comment (zellerh) [inline core/conn/security_dll/LICENSE]: Overall the changes look very good, my only question is about this deleted file. Could you explain why we were able to delete this?

**Comment by robertamarton** [comment]
This code originally was written for a different security implementation that included OpenSSL files in the source tree.  However, this code was subsequently rewritten that removed OpenSSL dependency. Therefore, Trafodion no longer has a source dependency with OpenSSL.  When changes were made to remove the OpenSSL dependency, the LICENSE file should have been removed but was not.  So, removing this file is a code cleanup operation.

**Comment by robertamarton** [comment]
jenkins, retest

**Comment by DaveBirdsall** [comment]
I am committing this change.

**Comment by DaveBirdsall** [comment]
It appears that I mistakenly merged this to the master. I will now commit this to release1.3.

**Pull request merged**